tools/zbkey

```diff
@@ -173,24 +183,20 @@ print "Sending data request packet..."
 #print kbutils.hexdump(data_assoc_packet)
 kb.inject(data_assoc_packet)
 
-# receiving the association response and setting the device_short_address
-value = None
-start = time.time()
-while (start+arg_delay > time.time()):		
-    # Does not block
+
+while True:
     recvpkt = kb.pnext()
-    # Check for empty packet (timeout) and valid FCS
-    if recvpkt != None and recvpkt['validcrc']:
```

Comment:
Why remove the crc check? If it's corrupted, the associate_response_handle is likely to have issues parsing or similar